platformcomponents/desktop/content-list-item.json

```diff
@@ -0,0 +1,25 @@
+{
+  "content-list-item": {
+    "comment": "Applied to file attachments from Content tab",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=49651%3A114181",
+    "iconForeground": "@theme-common-text-white",
```

Comment:
group this under 'previewbox'

Reply:
Pushed commit
[Grouped icon colors under previewBox key in content-file-list-item](https://github.com/momentum-design/tokens/pull/174/commits/a50f8c30e2dcda83a26ecc6af3600581344ffddf)
Changing this conversation to Resolved.